packages/world/mud.config.ts

```diff
@@ -77,19 +77,16 @@ export default defineWorld({
     // ------------------------------------------------------------
     ObjectTypeMetadata: {
```

Comment:
should we rename this to `ObjectMassEnergy` or something other than metadata so that we can use that for later